<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [All Test Suite]">
  <test thread-count="5" name="F:/PhysicianNetwork/PhysicianNetwork(failed)">
    <classes>
      <class name="Tests.Registration.RegistrationWithExistedEmailTest">
        <methods>
          <include name="exit"/>
          <include name="start"/>
          <include name="screenshotOnFailure"/>
          <include name="userCanRegisterSuccessfully"/>
        </methods>
      </class> <!-- Tests.Registration.RegistrationWithExistedEmailTest -->
      <class name="Tests.Registration.RegistrationSuccessfullyTest">
        <methods>
          <include name="userCanRegisterSuccessfully"/>
          <include name="exit"/>
          <include name="start"/>
          <include name="screenshotOnFailure"/>
        </methods>
      </class> <!-- Tests.Registration.RegistrationSuccessfullyTest -->
      <class name="Tests.Registration.RegistrationWithExistedPhoneNumberTest">
        <methods>
          <include name="exit"/>
          <include name="screenshotOnFailure"/>
          <include name="start"/>
          <include name="userCanRegisterSuccessfully"/>
        </methods>
      </class> <!-- Tests.Registration.RegistrationWithExistedPhoneNumberTest -->
      <class name="Tests.Login.LoginWithInvalidPassTest">
        <methods>
          <include name="exit"/>
          <include name="userCanNotLoginWithInvalidEmail"/>
          <include name="start"/>
          <include name="screenshotOnFailure"/>
        </methods>
      </class> <!-- Tests.Login.LoginWithInvalidPassTest -->
      <class name="Tests.Login.LoginWithInvalidEmailTest">
        <methods>
          <include name="exit"/>
          <include name="start"/>
          <include name="screenshotOnFailure"/>
          <include name="userCanNotLoginWithInvalidEmail"/>
        </methods>
      </class> <!-- Tests.Login.LoginWithInvalidEmailTest -->
      <class name="Tests.Login.LoginSuccessfullyTest">
        <methods>
          <include name="exit"/>
          <include name="userCanLoginWithValidEmailAndPassword"/>
          <include name="screenshotOnFailure"/>
          <include name="start"/>
        </methods>
      </class> <!-- Tests.Login.LoginSuccessfullyTest -->
    </classes>
  </test> <!-- F:/PhysicianNetwork/PhysicianNetwork(failed) -->
</suite> <!-- Failed suite [All Test Suite] -->
